src/net/sourceforge/kolmafia/session/ChoiceControl.java

```diff
@@ -2882,6 +2882,7 @@ else if (text.contains("takes your nifty new watch")) {
           if (NumberologyManager.numberologyPrize(result).startsWith("fight")) {
             KoLAdventure.clearLocation();
           }
+          break;
```

Comment:
move this out one level please.  The idiom seems to be

`case
{
}
break`
 whereas this looks to be the functionally equivalent 
`case
{
break
}
`